include/capstone/systemz.h

```diff
@@ -115,6 +87,34 @@ typedef enum sysz_reg {
 	SYSZ_REG_ENDING,
 } sysz_reg;
 
+// Instruction's operand referring to memory
+// This is associated with SYSZ_OP_MEM operand type above
+typedef struct sysz_op_mem {
+	sysz_reg base;		// base register
```

Comment:
same here and few other places: this will break bindings. what do you think about limiting the changes so that we are still compatible with bindings?